src/Icons/composer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "symfony/ux-icons",
+    "type": "symfony-bundle",
+    "description": "Twig component to use svg icons in your Symfony app",
+    "keywords": [
+        "symfony-ux",
+        "twig",
+        "icons"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Icons\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Icons\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=8.1",
```

Comment:
```suggestion
        "php": ">=8.2",
```

Reply:
Is this what we want?

Reply:
Symfony 7.x requires >= 8.2, but 6.4 still allows 8.1 

We can accept 8.1, but active support ended 2 months ago, and security updates will at the end of the year. 
During the experimental phase, not a problem of course :) 

So is there something in 8.2 we might need (readonly classes? not a real _need_ i think) ?

Reply:
I think requiring 8.1 makes this package consistent with the other packages. If we make 8.2, I think we'll need to make some considerations in the CI system that automatically runs tests for all components (I'd prefer not to do this).